web/impact/impact/tests/test_reserve_office_hour_view.py

```diff
@@ -259,14 +259,18 @@ def test_outlook_link_in_reserve_office_hour_email_to_mentor(self):
     def assert_response_contains_session_details(self, response, office_hour):
         office_hour.refresh_from_db()
         timecard = response.data['timecard_info']
+        calendar_data = timecard['calendar_data']
         if office_hour.startup:
             startup_name = office_hour.startup.organization.name
         else:
             startup_name = ""
+        calendar_data = timecard['calendar_data']
         oh_details = {'finalist_first_name': office_hour.finalist.first_name,
                       'finalist_last_name': office_hour.finalist.last_name,
                       'topics': office_hour.topics,
-                      'startup': startup_name}
+                      'startup': startup_name,
+                      "finalist_email": office_hour.finalist.email,
+                      "calendar_data": calendar_data}
         self.assertDictEqual(timecard, oh_details)
```

Comment:
This is a weird pattern - we construct `oh_details` using data from the `timecard` and then we compare the result to `timecard`. 

I understand the motivation - you want to use `assertDictEqual` and you need the calendar_data to match -  but inserting the data from the actual result into the expected result seems like it crosses a line.

Reply:
`calendar_data` returned from the Add2cal package has variables data like UUID's that will always return different values. Hence we can't get the same resulting from different calls. My intension here was to just test that the response contains the `calendar_data` key.